nikolina/package.json

```diff
@@ -4,16 +4,18 @@
   "description": "learning-js",
   "main": "index.js",
   "repository": "https://github.com/Pannonians/learning-js",
-  "author": "Petar Vukmanovic",
-  "license": "MIT",
   "scripts": {
     "start": "node index.js",
-    "inquirer": "yarn workspace nikolina-inquirer"
+    "inquirer": "yarn workspace nikolina-inquirer",
+    "explorer": "yarn workspace explorer"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "type": "commonjs",
-  "private":"true",
-  "workspaces": ["./inquirer"]
+  "private": true,
+  "workspaces": [
+    "./inquirer",
+    "./explorer"
+  ]
```

Comment:
Please revert these changes. Please don't make any changes to files outside of the scope, and the scope was to scaffold project.